docs - add install step for fingerprint verification

lisakowen opened the pull request:
restructure the "install greenplum" topic to include tasks for downloading/obtaining the software and verifying the SHA.  some content has been conditionalized.

doc review site link (behind VPN):  https://docs-lisa-install-verify.sc2-04-pcf1-apps.oc.vmware.com/7-0/install_guide/install_gpdb.html

also attaching a pdf for oss doc changes:
[install_oss.pdf](https://github.com/greenplum-db/gpdb/files/7341747/install_oss.pdf)

mperezfuster commented on gpdb-doc/dita/install_guide/verify_sw.xml:
```diff
@@ -0,0 +1,39 @@
+    <p>After you download a Greenplum Database server or component software package,
+      you can verify the integrity of the software as follows:</p>
+    <ol>
+      <li>On the VMware Tanzu Greenplum <i>Release Download Files</i> page, click the
```
maybe this location could be made a bit clearer, it took me a while to realise I had to select the product to see the i icon.

lenapivotal replied:
I agree, we could mention the word "dropdown" somewhere as it's not obvious where the "I" is ....

mperezfuster reviewed (approved):
thanks for creating this! I only had a small comment

lenapivotal reviewed (approved):
LGTM

lisakowen commented:
thanks for reviewing @mperezfuster and @lenapivotal .  i was assuming that the user had just downloaded the package and was already on the appropriate page (Release Download Files).  given that might not be the case, i just added an explicit step to locate the package.

dyozie commented:
Can you check out the conflict for the install guide file? I was thinking you could use something like the modified text for download install, that covers both open source and proprietary versions.  Also, it would be great if for the new verifying topic if you could remove the condition but add (VMware Tanzu SQL) at the end of the title.

For context, I checked in some dita cleanup prior to converting to markdown which edited some passages to remove use of conditional text in the install guide.

lisakowen commented:
@dyozie - i fixed the merge conflict (removed conditionals, removed oss-specific topic).  also renamed the verify topic and removed an unrelated condition.  let me know if you were looking for something different.

dyozie commented:
Thanks @lisakowen - that looks great!
